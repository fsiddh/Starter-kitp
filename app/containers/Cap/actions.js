import * as types from './constants';

export function authenticate(user) {
  return {
    type: types.LOGIN_REQUEST,
    user,
  };
}

export function logout() {
  return {
    type: types.LOGOUT_REQUEST,
  };
}

export function changeOrg(orgID, successCallback) {
  return {
    type: types.SWITCH_ORG_REQUEST,
    orgID,
    successCallback,
  };
}

export function getUserData() {
  return {
    type: types.GET_USER_DATA_REQUEST,
  };
}

export function getSidebarMenuData() {
  return {
    type: types.GET_SIDEBAR_MENU_DATA_REQUEST,
  };
}
