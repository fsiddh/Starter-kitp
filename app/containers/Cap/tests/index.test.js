import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars
import { Cap } from '../index';
import * as actions from '../actions';

describe('<Cap />', () => {
  it('renders <Cap /> correctly', () => {
    const props = {
      match: {
        path: '',
      },
      location: {
        search: '',
      },
      Global: {
        token: '',
      },
      actions,
    };
    const tree = renderer.create(<Cap {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
