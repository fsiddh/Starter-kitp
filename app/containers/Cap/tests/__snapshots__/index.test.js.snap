// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cap /> renders <Cap /> correctly 1`] = `null`;
